app/views/standup_meeting_groups/_standup_meeting_group.html.erb

```diff
@@ -1,9 +1,44 @@
-<div id="<%= dom_id standup_meeting_group %>">
-  <h2><%= link_to standup_meeting_group.name, standup_meeting_group %></h2>
-
-  <p>
-    <%= link_to "Edit", edit_standup_meeting_group_path(standup_meeting_group) %>
-  </p>
-
-  <hr>
+<div class="max-w-6xl mx-auto p-4">
+  <div class="text-sm breadcrumbs max-w-xs">
+    <ul>
+      <li><%= link_to "Standup Groups", standup_meeting_groups_path %></li> 
+      <li><%= standup_meeting_group.name %></li> 
+    </ul>
+  </div>
+  <div class="py-4 flex justify-between">
+    <h1 class="text-3xl font-bold text-left mb-4 md:mb-0"><%= standup_meeting_group.name %></h1>
+    <%= button_to 'Leave', standup_meeting_group_join_path(standup_meeting_group, standup_meeting_group_user), method: :delete, data: { turbo: false }, class: 'link mt-4' %>
+  </div>
+  <%= render StandupMeeting::CheckInCtaComponent.new(standup_meeting_group: standup_meeting_group, standup_meeting: current_user_standup_meeting, current_user:) %>
+  <div class="divider my-4 "></div>
+  <div class="flex flex-col">
+    <div class="flex flex-col md:flex-row justify-between">
+      <h1 class="text-2xl text-center md:text-left">Previous Standups</h1>
+      <%= render StandupMeeting::DateNavigatorComponent.new(
+        standup_meeting_group: standup_meeting_group
+      ) %>
```

Comment:
I don't think we want this in the show? The behavior feels off (when you click on the left/right arrow it redirects to the standup_meetings index.

Reply:
I agree we shouldn't have the arrows -- maybe we can add a flag to show next navigation?

Reply:
nix on the flag -- tends to be a smell, anyway. I extracted the date jumping functionality into its own component and am using only the date jumper component on the show page.